dash_labs/plugins/pages.py

```diff
@@ -248,6 +261,15 @@ def plug(app):
                     page_module, "layout"
                 )
 
+
+def plug(app):
+    dash.page_registry = OrderedDict()
+
+    if os.path.exists("pages"):
+        _import_layouts_from_pages()
+    else:
+        print("A folder called `pages` does not exist.")
```

Comment:
This is also an older version.  There is now a [warning]( https://github.com/plotly/dash-labs/blob/main/dash_labs/plugins/pages.py#L321) since it's possible to create a multi page app without a `pages` folder - just by passing layout= directly into register_page.